config/internal/ml-metadata/metadata-grpc.networkpolicy.yaml.tmpl

```diff
@@ -0,0 +1,27 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: ds-pipeline-metadata-grpc-{{ .Name }}
+  namespace: {{ .Namespace }}
+spec:
+  podSelector:
+    matchLabels:
+      app: ds-pipeline-metadata-grpc-{{ .Name }}
+      component: data-science-pipelines
+  ingress:
+    - ports:
```

Comment:
should we add these ports to ingress too maybe? https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/config/internal/ml-metadata/metadata-envoy.service.yaml.tmpl#L12
Looks like the envoy and other mlmd services are utilizing them.

Reply:
Nope. We have these two flows:

`stuff --1--> envoy --2--> mlmd`
`stuff --3--> mlmd`

Things that connect to envoy (arrow 1) need to connect on those two ports you linked to.
Envoy itself only needs to connect to 8080 on mlmd (arrow 2).
Things that connect to mlmd directly only need to connect to 8080 on mlmd (arrow 3).

Arrows 2 and 3 are the ones we care about in this NetworkPolicy, so 8080 is all we need.